webapp/channels/src/components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -434,9 +442,16 @@ export default class GroupsList extends React.PureComponent<Props, State> {
             filterIsUnlinked: false,
         };
         this.setState(newState as any);
-        this.props.actions.getLdapGroups(this.state.page, LDAP_GROUPS_PAGE_SIZE, {q: ''}).then(() => {
-            this.setState({loading: false});
-        });
+        this.props.actions.getLdapGroups(this.state.page, LDAP_GROUPS_PAGE_SIZE, {q: ''}).then(this.handleGetGroupsResponse);
+    };
+
+    handleGetGroupsResponse = (response: any) => {
+        if (response.error) {
```

Comment:
the conditional part could better be defined in a simpler way as:  `this.setState({fetchError: Boolean(response.error)});`